src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,36 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../api/types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export const TodoInfo: React.FC<Props> = ({
+  todo: {
+    id,
+    title,
+    completed = false,
+    user,
+  },
+}) => {
+  return (
+    <article
+      data-id={id}
+      className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
+      )}
+    >
+      <h2 className="TodoInfo__title">
+        {title}
+      </h2>
+      {user
+        && (
+          <UserInfo
+            user={user}
+          />
```

Comment:
```suggestion
          <UserInfo user={user} />
```
You can make it in one line if the amount of props is less than 3